src/Avalonia.Controls/Presenters/ScrollContentPresenter.cs

```diff
@@ -360,12 +362,17 @@ private void UpdateScrollableSubscription(IControl child)
                             .Subscribe(x => scrollable.CanVerticallyScroll = x),
                         this.GetObservable(OffsetProperty)
                             .Skip(1).Subscribe(x => scrollable.Offset = x),
-                        Disposable.Create(() => scrollable.InvalidateScroll = null));
+                        Disposable.Create(() => scrollable.ScrollInvalidated -= ScrollInvalidated));
```

Comment:
FYI: `Disposable.Create` has a variant that takes a parameter which allows you to easily drop allocations (lambda and closure) :)

Reply:
TBH that whole block could probably be done without Rx but I was just refactoring the minimum to reduce noise in this PR...